public/react/Configure/Teams/components/TextField/Textfield.tsx

```diff
@@ -0,0 +1,47 @@
+import type { FC } from "react";
+import type { DefaultFormFields, FieldUpdates } from "../../ConfigureTeams";
+
+import React from "react";
+
+import AtlasTextField from "@atlaskit/textfield";
+import { UseFormReturn } from "react-hook-form";
+import { Field } from "@atlaskit/form";
+
+function isFieldUpdate<TProperty extends keyof DefaultFormFields>(event: {
+  name: string;
+}): event is FieldUpdates<TProperty> {
+  return ["sprintLength"].includes(event.name);
```

Comment:
Are you initially implementing the `Textfield` component to only work for `ConfigureTeams` and more specifically the `sprintlength`field and then only refactor it to a more generic `Textfield` if its going to be used by more components in the future?

Reply:
Yeah kinda. I plan on keeping it detailed specific right now since the primary focus is the storage service layer. A generic one in the future might be nice though